src/vs/workbench/contrib/scm/electron-browser/scmViewlet.ts

```diff
@@ -884,10 +884,20 @@ export class RepositoryPanel extends ViewletPanel {
 			new ResourceRenderer(this.listLabels, actionItemProvider, () => this.getSelectedResources(), this.themeService, this.menus)
 		];
 
-		this.list = this.instantiationService.createInstance(WorkbenchList, this.listContainer, delegate, renderers, {
-			identityProvider: scmResourceIdentityProvider,
-			keyboardNavigationLabelProvider: scmKeyboardNavigationLabelProvider
-		}) as WorkbenchList<ISCMResourceGroup | ISCMResource>;
+		const horizontalScrollingSCM = this.configurationService.getValue('workbench.tree.horizontalScrollingSCM') || 'inherit';
```

Comment:
I think that the ending of this line could be stripped: `|| 'inherit'`. That should be the default setting. I think the only case when you would need this additional check is when someone introduces a breaking change (changing the key of the setting for example), but that should be detected early on.